lib/main.js

```diff
@@ -77,6 +78,9 @@ function initSession(shiny, options, shinyServer) {
     if (options.reconnect) {
       factory = reconnect.decorate(factory, options);
     }
+    if (!options.reconnect) {
```

Comment:
This is SUCH a nitpick, but can you make this an `else` so it's clear this one and the reconnect decorator are substitutes? Other than that, LGTM!